test/e2e/k8s_util.go

```diff
@@ -751,7 +754,7 @@ func (k *KubernetesUtils) Validate(allPods []Pod, reachability *Reachability, po
 		}
 		reachability.Observe(r.podFrom, r.podTo, r.connectivity)
 		if r.connectivity != Connected && reachability.Expected.Get(r.podFrom.String(), r.podTo.String()) == Connected {
-			log.Warnf("FAILED CONNECTION FOR ALLOWED PODS %s -> %s !!!! ", r.podFrom, r.podTo)
+			log.Warnf("FAILED CONNECTION FOR ALLOWED PODS %s -> %s ON %s !!!!", r.podFrom, r.podTo, protocol)
```

Comment:
did you mean to print the port here, and not the protocol?

Reply:
Oh, my bad. That's used for my dev debug. I will remove it.